test/metabase/db/custom_migrations_test.clj

```diff
@@ -1895,10 +1895,10 @@
                            (map (fn [card] (update card :visualization_settings #(json/parse-string % keyword)))))]
             (doseq [{:keys [name] :as card} cards]
               (testing (format "Migrating a card where: %s" name)
-                (is (= (-> (get-in area-bar-combo-cards-test-data [name :expected])
-                           (dissoc :name))
-                       (-> card
-                           (dissoc :name))))))))))))
+                (is (partial= (-> (get-in area-bar-combo-cards-test-data [name :expected])
```

Comment:
Probably better to use `=?` instead of `partial=`, `partial=` is basically deprecated at this point since `=?` does everything it does and more